app/scripts/actions/ForceDownload.js

```diff
@@ -0,0 +1,162 @@
+//download.js v4.2, by dandavis; 2008-2016. [CCBY2] see http://danml.com/download.html for tests/usage
+// v1 landed a FF+Chrome compat way of downloading strings to local un-named files, upgraded to use a hidden frame and optional mime
+// v2 added named files via a[download], msSaveBlob, IE (10+) support, and window.URL support for larger+faster saves than dataURLs
+// v3 added dataURL and Blob Input, bind-toggle arity, and legacy dataURL fallback was improved with force-download mime and base64 support. 3.1 improved safari handling.
+// v4 adds AMD/UMD, commonJS, and plain browser support
+// v4.1 adds url download capability via solo URL argument (same domain/CORS only)
+// v4.2 adds semantic variable names, long (over 2MB) dataURL support, and hidden by default temp anchors
+// https://github.com/rndme/download
+
+(function(root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    // AMD. Register as an anonymous module.
+    define([], factory)
+  } else if (typeof exports === 'object') {
+    // Node. Does not work with strict CommonJS, but
+    // only CommonJS-like environments that support module.exports,
+    // like Node.
+    module.exports = factory();
+  } else {
+    // Browser globals (root is window)
+    root.download = factory();
+  }
+}(this, function () {
+
+  return function download(data, strFileName, strMimeType) {
+
+    var self = window, // this script is only for browsers anyway...
+      defaultMime = "application/octet-stream", // this default mime also triggers iframe downloads
+      mimeType = strMimeType || defaultMime,
+      payload = data,
+      url = !strFileName && !strMimeType && payload,
+      anchor = document.createElement("a"),
+      toString = function(a){return String(a);},
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Variable 'toString' has the same name as the function declared in line undefined.](https://www.codacy.com/app/jlcfigueira/hub-client/file/2683057581/issues/source?bid=3141166&fileBranchId=3335842#l33)